boefjes/boefjes/app.py

```diff
@@ -241,6 +260,72 @@ def _start_working(
                     # The docker runner could have handled this already
                     scheduler_client.patch_task(p_item.id, status)  # Note that implicitly, we have p_item.id == task_id
                     logger.info("Set status to %s in the scheduler for task[id=%s]", status, p_item.data.id)
+
+                if not isinstance(handler, BoefjeHandler) or not duplicated_items:
+                    # We do not deduplicate normalizers
+                    continue
+
+                if out is None:
+                    # `out` will be None on failures or for Docker boefjes (until #4304 is merged), in which case there
+                    # is nothing to save to Bytes and the statuses have been patched in the except block.
+                    for item in duplicated_items:
+                        # Instead of duplicating errors, give the other deduplicated tasks another chance
+                        scheduler_client.patch_task(item.id, TaskStatus.QUEUED)
```

Comment:
Not sure if this is needed. It would be weird to have the first one as failed, but the remaining tasks complete. Why not retry the first one, and duplicate those results up to N times?